commit bf324e3440246a487997a1624ac862e3e4027f9e
Author: Michael Olbrich <m.olbrich@pengutronix.de>
Date:   Mon Aug 4 19:05:18 2014 +0200

    add missing include
    
    Without this building without HAVE_VA_X11 fails with:
    [...]
    In file included from i965_drv_video.c:37:0:
    i965_output_wayland.h:31:26: error: unknown type name 'VADriverContextP'
    i965_output_wayland.h:34:31: error: unknown type name 'VADriverContextP'
    i965_drv_video.c:5243:9: error: 'i965_output_wayland_init' undeclared here (not in a function)
    i965_drv_video.c:5244:9: error: 'i965_output_wayland_terminate' undeclared here (not in a function)
    [....]

diff --git a/src/i965_output_wayland.h b/src/i965_output_wayland.h
index 61ca39f..acea8d9 100644
--- a/src/i965_output_wayland.h
+++ b/src/i965_output_wayland.h
@@ -26,6 +26,7 @@
 #define I965_OUTPUT_WAYLAND_H
 
 #include <stdbool.h>
+#include <va/va_backend.h>
 
 bool
 i965_output_wayland_init(VADriverContextP ctx);
